terraform/modules/daos_server/templates/certs_install.inc.sh.tftpl

```diff
@@ -17,7 +17,7 @@ get_ca_from_sm() {
 
   # Loop until the secret exists.
   # If the secret is not found in max_secret_wait_time, then exit.
-  max_secret_wait_time="5 mins"
+  max_secret_wait_time="10 mins"
```

Comment:
I would prefer to investigate why these timeout, rather than increasing this time. 
Maybe we should create a follow-up task to review this? 
One thing comes to my mind is that the secret may not be created before the servers are, we should make sure the dependency exists in the terraform so the machines are only created after the secret.

Reply:
to be clear, I am not suggesting we keep 5. We should keep 10 and then create the appropriate task to reduce it back to 5 or less.